mslib/utils/mssautoplot.py

```diff
@@ -421,12 +433,16 @@ def draw(self, flight, section, vertical, filename, init_time, time, url, layer,
 
 class LinearViewPlotting(Plotting):
     # ToDo Implement access of MSColab
-    def __init__(self, cpath):
+    def __init__(self, cpath, msc_url, msc_auth_password, msc_username, msc_password):
         super(LinearViewPlotting, self).__init__(cpath)
```

Comment:
comparing with class TopViewPlotting(Plotting):

We have there super(TopViewPlotting, self).__init__(cpath, msc_url, msc_auth_password, msc_username, msc_password) too.

Is that required? If not please look if it is in TopViewPlotting required too or could be changed to a similiar.

Reply:
Yes that is required. Changes Done.